src/lib.rs

```diff
@@ -225,6 +233,10 @@ pub enum PodCastError {
 }
 impl core::fmt::Display for PodCastError {
   fn fmt(&self, f: &mut core::fmt::Formatter) -> core::fmt::Result {
+    // Currently, this is unlikely to work for the spirv targets
+    #[cfg(target_arch = "spirv")]
+    write!(f, "PodCastError", self);
```

Comment:
the `,self` part should be removed here. There's no interpolation positions within the literal.

Reply:
Hmm, I'm surprised that this compiled.

But yes, well spotted - fixed.